cpp/src/aztec/plonk/proof_system/verifier/verifier.test.cpp

```diff
@@ -298,6 +298,7 @@ TEST(verifier, verify_arithmetic_proof_small)
     EXPECT_EQ(result, true);
 }
 
+// TODO(Cody): This test is suddenly very slow to run?
```

Comment:
Not sure why this would've changed but this is constructing a proof of non-trivial size so makes sense that it would take some time I suppose. Prover stuff here could be mocked like I was planning to do for the equivalent honk verifier test.

Reply:
Makes sense, just thought I'd flag the change for a time in case we noticed some of our tests were unexpectedly slow--maybe this would give a clue.